eth2util/keystore/keystore.go

```diff
@@ -62,6 +62,33 @@ func StoreKeys(secrets []*bls_sig.SecretKey, dir string) error {
 	return storeKeysInternal(secrets, dir, "keystore-%d.json")
 }
 
+// KeymanagerReq represents the keymanager API request body for POST request. Refer: https://ethereum.github.io/keymanager-APIs/#/Local%20Key%20Manager/importKeystores
+type KeymanagerReq struct {
+	Keystores []keystore `json:"keystores"`
+	Passwords []string   `json:"passwords"`
+}
+
+// KeymanagerReqBody constructs a KeymanagerReq using the provided secrets and returns it.
+func KeymanagerReqBody(secrets []*bls_sig.SecretKey) (KeymanagerReq, error) {
+	var resp KeymanagerReq
+	for _, secret := range secrets {
+		password, err := randomHex32()
+		if err != nil {
+			return KeymanagerReq{}, err
+		}
+
+		store, err := encrypt(secret, password, rand.Reader)
```

Comment:
On second thought, rather just export Encrypt, and do this API stuff in cmd package, this shouldn't be part of this package.

Reply:
Agreed, will do this refactor in a separate PR